### PR TITLE
Day19 Activities

### PR DESCRIPTION
**Pull request by sharonjohnson007**
please review sir

### CODE REVIEW

File: Day19/src/main/java/ComputePI.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                System.out.println("Impossible!!!");
+            }
+        }
+        System.out.println("Pi values using different MAX_DENOMINATOR: " + sum * 4);
+        return sum * 4;
+    }
+
+    //To calculate using MAX_TERM
+    public static double computePiUsingMAX_TERM(double MAX_TERM) {
+        double sum1 = 0.0;
```
Comment (dorarajgcit) [inline Day19/src/main/java/ComputePI.java]: why the variable is named as sum1 ?

File: Day19/src/main/java/ComputePI.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                sum1 -= 1.0 / (term * 2 - 1);
+            }
+        }
+        System.out.println("Pi values using different MAX_TERM: " + sum1 * 4);
+        return sum1 * 4;
+    }
+
+    //Pi values of built-in double constant Math.PI
+    public static void mathPi(double value) {
+        double sum2 = Math.PI;
```
Comment (dorarajgcit) [inline Day19/src/main/java/ComputePI.java]: What is the use of this method?
What is the purpose of passing value to this method?
What is the purpose of the variable sum2 and why it is named as sum2 ?

File: Day19/src/main/java/ComputePI.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        System.out.println("Percents of MathPi with MAX_TERM: " + calculatedPercent(result8));
+
+    }
+
+    //To calculate using MAX_DENOMINATOR
+    public static double findPi(double MAX_DENOMINATOR) {
+        double sum = 0.0;
+        for (double denominator = 1; denominator <= MAX_DENOMINATOR; denominator += 2) {
+            if (denominator % 4 == 1) {
+                sum = sum + (1 / denominator);
```
Comment (dorarajgcit) [inline Day19/src/main/java/ComputePI.java]: Use compound assignment operator

File: Day19/src/main/java/Fibonacci.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        System.out.println("The first " + nMax + " Fibonacci numbers are:");
+        System.out.print("1" + " " + "1");
+        //To find fibonacci series
+        while (n <= nMax) {
+            fn = fnMinus1 + fnMinus2;
+            n++;
+            fnMinus2 = fnMinus1;
+            fnMinus1 = fn;
+            System.out.print("\t" + fn);
+            sum = sum + fn;
```
Comment (dorarajgcit) [inline Day19/src/main/java/Fibonacci.java]: Use compound assignment operator

File: Day19/src/main/java/Tribonacci.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.print("1" + " " + "1" + " " + "2");
+        //To find tribonacci series
+        while (n <= nMax) {
+            tn = fnMinus1 + fnMinus2 + fnMinus3;
+            fnMinus1 = fnMinus2;
+            fnMinus2 = fnMinus3;
+            fnMinus3 = tn;
+            ++n; //5
+            System.out.print("\t" + tn);
+            sum = sum + tn;
```
Comment (dorarajgcit) [inline Day19/src/main/java/Tribonacci.java]: Use compound assignment operator

**Review by dorarajgcit — CHANGES_REQUESTED** [review]
Left few comments, take a look

File: Day19/src/main/java/ComputePI.java (hunk 2 above)
Comment (sharonjohnson007) [reply Day19/src/main/java/ComputePI.java]: sir to show the difference between the other pi values and Math.PI values for comparison
before adding the methods i used the sum2 to use it in one class without using methods i will remove it sir

**Pull request merged**